### DISCUSSION TITLE
淘宝小程序内load gltf use draco extension报错

### DISCUSSION THREAD

**Discussion by rebeccaKang** [galacean/discussion#82] [Mini Program（小程序）]
淘宝小程序用官网快速生成的demo，加载没压缩的普通gltf模型没问题，加载一个用draco压缩的glb模型，报错
<img width="438" alt="image" src="https://github.com/galacean/engine/assets/1732199/7e854be5-5c3f-433b-b970-da5af575470b">

<img width="607" alt="image" src="https://github.com/galacean/engine/assets/1732199/b2a06e61-7f3d-4412-a1e0-65ce04fa1fe1">

ts里是有worker的，但非web环境好像用不了worker
请问现在这情况我该怎么解决？

**Comment by eyworldwide** [comment]
@rebeccaKang draco 使用到了 WebAssembly 能力，请先确认一下淘宝小程序是否支持 WebAssembly。

**Comment by eyworldwide** [comment]
这个问题跟引擎无关，从 https://github.com/galacean/engine/issues/1583 转移到了这里
